File: Server/DataAccess/Entities/Dish.cs
namespace DataAccess.Entities
{
    public class Dish : Entity
    {
        public decimal Price { get; set; }
        public decimal Portion { get; set; }
        public string Name { get; set; } = null!;
        public string Description { get; set; } = null!;
        public string Type { get; set; } = null!;

        public int CateringId { get; set; }
        public virtual Catering Catering { get; set; } = null!;

        public virtual List<OrderDish> OrderDishes { get; set; } = null!;

        public override string? GetOwnerId() => Catering?.GetOwnerId();
    }
}

File: mobile/app/src/main/java/com/geneirodan/nextgenmeal/data/Dish.kt
package com.geneirodan.nextgenmeal.data

import androidx.annotation.Keep
import kotlinx.serialization.Serializable

@Serializable
@Keep
data class Dish(
    val id: Int,
    val price: Double,
    val portion: Double,
    val name: String,
    val description: String,
    val type: String,
    val cateringId: Int
)